staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -456,6 +456,15 @@ func isDryRun(url *url.URL) bool {
 	return len(url.Query()["dryRun"]) != 0
 }
 
+// TODO: currently strict validation is set via
+// query param. Alternatively we could use a request header.
```

Comment:
Query param is good (easier to use from some languages).

Reply:
Agreed. Removed the TODO about this.